org.freedesktop.dabrain34.squared.json

```
@@ -0,0 +1,46 @@
+{
+  "app-id": "org.freedesktop.dabrain34.squared",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "squared",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=puseaudio",
```

```suggestion
    "--socket=pulseaudio",
```

flatpak-builder should have thrown an error at that point.